### PR TITLE
`<MultiSelect/>` - fix disabled state

### PR DESCRIPTION
**Pull request by TalZa1810**
### 🔦 Summary

https://github.com/wix-private/wsr-issues/issues/238 

### ✅ Checklist
- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Other (explain)
- 🔬 Change is tested
  - [x] Component
  - [x] Visual test

### CODE REVIEW

**Comment by TalZa1810** [comment]
I've created 2 unit tests to check that there's no option to choose any of the options once the input is disabled (both for keyboard and mouse). Moreover, another 2 unit tests to make sure that they have 'disabled' attribute for MultiSelect with & without tags. The e2e test it only for the snapshot. 
Is it enough? WDYT?

**Comment by TalZa1810** [comment]
According to Zeplin, the Tag should always have a remove button. When it is disabled and when it is not disabled. Therefore, I'm not sure if this test is accurate:
"Tag should have not remove button if disabled is true".

[Zeplin link](https://app.zeplin.io/project/5864e02695b5754a69f56150/screen/58932bf3fcb39ff04c7179a2)

File: src/MultiSelect/InputWithTags.js [hunk 1]
```diff
@@ -26,7 +26,9 @@ class InputWithTags extends React.Component {
 
   handleClick(e) {
     this.input.focus();
```
Comment (mosheKerbel) [inline src/MultiSelect/InputWithTags.js]: do we want it to get focused upon click if it is disabled?

File: src/MultiSelect/MultiSelect.spec.js [hunk 2]
```diff
@@ -153,6 +153,26 @@ describe('MultiSelect', () => {
       expect(driver.inputWrapperHasError()).toBeTruthy();
     });
 
+    it('should have disabled attribute on input if disabled', async () => {
+      const { driver } = createDriver(
+        <MultiSelect disabled options={options} />,
+      );
+      expect(driver.isDisabled()).toBe(true);
```
[the remark this answers is not in the document]
Comment (TalZa1810) [reply src/MultiSelect/MultiSelect.spec.js]: The `isDisbaled` is to check the input itself. I've added another function `inputWrapperDisabled` to check if the inputWrapper div has a "disabled" classname. The disable attribute is added only for the input itself and not to the wrapper.

[the remark this answers is not in the document]
Comment (shlomitc) [reply src/MultiSelect/MultiSelect.spec.js]: cool. can you rename it to `inputWrapperIsDisabled`?

[the remark this answers is not in the document]
Comment (TalZa1810) [reply src/MultiSelect/MultiSelect.spec.js]: Done
@mosheKerbel

File: src/Tag/Tag.js [hunk 3]
```diff
@@ -43,11 +44,11 @@ class Tag extends WixComponent {
 
   _renderRemoveButton() {
     const { removable, disabled, size } = this.props;
-    if (removable && !disabled) {
+    if (removable) {
       return (
         <CloseButton
           size={size === 'large' ? 'medium' : 'small'}
-          skin="dark"
+          disabled={disabled}
```
Comment (shlomitc) [inline src/Tag/Tag.js]: removing the `skin="dark"` changed the design of it
in this PR:
image

production:
image

according to the [specs](https://zpl.io/am4pwo3) it should look like what's in production

**Comment by shlomitc** [comment]
@TalZa1810 - please fix according to Lital's feedback: https://github.com/wix/wix-style-react/pull/3201#pullrequestreview-222091056

**Comment by TalZa1810** [comment]
@shlomitc 
Done

**Comment by shlomitc** [comment]
it still looks like incorrect styles to me
current:
image

wanted:
image

@TalZa1810 - can you please sit together with @LittleTal and verify it one-on-one?

**Pull request merged**